testsuite/openshift/objects/gateway_api/route.py

```diff
@@ -73,7 +73,14 @@ def remove_all_hostnames(self):
         self.model.spec.hostnames = []
 
     @modify
-    def set_match(self, backend: "Httpbin", path_prefix: str = None):
+    def set_path_match(self, path_prefix: str):
+        """TODO"""
+        self.model.spec.rules.append(
```

Comment:
`set_backend_match` wouldnt work?

Reply:
It will, but I'm not using requests to this endpoint to reach the backend, only requests for 500 and 403, so it is redundant in my opinion. It is also possible to extend the backend match method below, but I wanted to introduce a different interface for this